docs/integrations/sources/mysql.md

```diff
@@ -20,8 +20,8 @@ If you can't find the data type you are looking for or have any problems feel fr
 | `bigint` | number |  |
 | `binary` | string |  |
 | `blob` | string |  |
-| `date` | string |  |
-| `datetime` | string |  |
+| `date` | string | MySql JDBC driver has limitation. ZERO-DATE value is forbidden. |
```

Comment:
I think from a UX point of view , Airbyte not supporting zero date is unfortunately not good enough as it will cause lots of hard-to-predict failures. Users are going to have zero dates in their DBs and don't always have agency in fixing them e.g: maybe they don't own the database and can't edit it. 

why don't we cast zero date values to null e.g: https://stackoverflow.com/questions/11133759/0000-00-00-000000-can-not-be-represented-as-java-sql-timestamp-error? 

in that case I would also love to add them to the test suite